src/Orchard.Environment.Extensions/Loaders/DynamicExtensionLoader.cs

```diff
@@ -66,7 +66,7 @@ public class DynamicExtensionLoader : IExtensionLoader {
                 }
                 catch (System.Exception ex) {
                     _logger.LogError(string.Format("Error trying to load extension {0}", descriptor.Id), ex);
-                    throw;
+                    return null;
```

Comment:
@Jetski5822 not sure, haven't checked this in a while, just did this so it keeps working if logger module was not available